### PR TITLE
Fixes and improvements

### PR DESCRIPTION
**Pull request by frankrousseau**
Hello,

  I fixed several things and did some improvements. Here they are (sorry for the big PR):
- Change icon (save changes and delete picture) to make them more meaningful and consistent with the other UIs.
- Fix rotation storage: translate 'LeftBottom' to 8 code (means portrait picture).
- Save photo browsing in the history
- Add a delete picture button in the picture navigation
- Add missing hover effect (orange color)
- Fix clearance locale

### CODE REVIEW

**Comment by aenario** [comment]
is this PR ready ?

**Comment by frankrousseau** [comment]
Yes I'm waiting for a review.

**Comment by aenario** [comment]
reviewing now

File: client/app/views/galery.coffee [hunk 1]
```diff
@@ -181,7 +195,16 @@ module.exports = class Galery extends ViewCollection
...
-        document.getElementById('uploader').addEventListener 'change', @onFilesChanged
+        element = document.getElementById('uploader')
+        element.addEventListener 'change', @onFilesChanged
+
+    # When trash button is clicked it proposes to delete the currently
+    # displayed picture. It asks for a confirmation before
+    onTrashClicked: =>
+      if confirm t 'photo delete confirm'
+          photo = @collection.get(@getIdPhoto())
+          console.log photo
```
Comment (aenario) [inline client/app/views/galery.coffee]: debug logging

File: client/vendor/scripts/photobox.js [hunk 2]
```diff
@@ -557,13 +557,14 @@
...
 	function updateIndexes(idx){
 		lastActive = activeImage;
-        if (idx < 0) {
-          idx = -1;
-        }
-        activeImage = idx;
-        activeURL = images[idx][0];
-        prevImage = (activeImage || (options.loop ? images.length : 0)) - 1;
-        nextImage = ((activeImage + 1) % images.length) || (options.loop ? 0 : -1);
+    if (idx < 0) {
```
Comment (aenario) [inline client/vendor/scripts/photobox.js:560]: indentation ?

File: server/controllers/photo.coffee [hunk 3]
```diff
@@ -121,10 +121,13 @@ module.exports.create = (req, res, next) =>
                 next err
             else
                 # Add date and orientation from EXIF data.
+                req.body.orientation = 1
                 if metadata?.exif?.orientation?
-                    req.body.orientation = metadata.exif.orientation
-                else
-                    req.body.orientation = 1
+                    if metadata.exif.orientation is 'LeftBottom'
```
Comment (aenario) [inline server/controllers/photo.coffee]: What about all other types of exif orientations ?

File: server/models/cozy_instance.coffee [hunk 4]
```diff
@@ -29,4 +29,4 @@ CozyInstance.getURL = (callback) ->
             .replace('https://', '')
             callback null, "https://#{url}/"
         else
-            callback new Error 'No instance domain set'
+            callback null, "http://localhost:9119/"
```
Comment (aenario) [inline server/models/cozy_instance.coffee]: isnt this for debug only ?

**Comment by frankrousseau** [comment]
Thank you for your comments. Here are the fixes.

**Comment by aenario** [comment]
Perfect. build at 93a6731

**Comment by frankrousseau** [comment]
Great, thank you!

**Pull request merged**